eng/.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why json vs yml? We should try to be consistent on that.

Reply:
There's no reason to use the different formats, they were just implemented differently in each repo.  I will create separate PRs for each repo to align the prettier configs.